tsp-output/@azure-tools/typespec-csharp/src/Generated/Audio.cs

```diff
@@ -259,8 +258,8 @@ public virtual async Task<ClientResult<CreateTranslationResponse>> CreateTransla
         {
             Argument.AssertNotNull(audio, nameof(audio));
 
-            (BinaryContent content, string contentType, RequestOptions options) = await audio.ToMultipartContentAsync().ConfigureAwait(false);
-            ClientResult result = await CreateTranslationAsync(content, contentType, options).ConfigureAwait(false);
+            using MultipartFormDataBinaryContent content = audio.ToMultipartContent();
```

Comment:
TODO: add wrapper type currently defined in https://github.com/Azure/azure-sdk-for-net/pull/42759 to emitted types

Reply:
Addressed

Reply:
Addressed